src/Makefile.mk

```diff
@@ -193,7 +193,7 @@ OFILES_KS = klh10.o prmstr.o fecmd.o feload.o wfio.o osdsup.o \
 	vmtape.o dvtm03.o	\
 	dvlhdh.o dvdz11.o dvch11.o \
 	dpsup.o \
-	dvhost.o dvlites.o
+	dvhost.o #dvlites.o is NOT needed for the KS-10
 
```

Comment:
YES. This we can do. If somebody adds lights to the KS (is that even sensible?) it is easy enough to add it again.